packages/InteractiveProfilingTool-Core.package/IPTModel.class/instance/shouldContinueProfiling..st

```diff
@@ -0,0 +1,3 @@
+accessing
+shouldContinueProfiling: aBoolean
```

Comment:
@tillprochaska I thought we deleted ```shouldContinueProfiling``` in favor of ```isCurrentlyProfiling```?
@twollnik @Alexander-Dubrawski why Do you have both?

__Seems as if this should be deleted here too__

Reply:
@linusha: I think the branch doesn’t have the latests changes from 22, so this will probably be solved after pulling in the latest change.